### PR TITLE
WIP: ask backend url for production in the task `grunt build`

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#34 

screen shot 2015-01-22 at 10 28 49 am

### CODE REVIEW

File: app/templates/client/Gruntfile.js [hunk 1]
```diff
@@ -421,13 +426,34 @@ module.exports = function (grunt) {
...
+          remote: 'https://github.com/.git',
           branch: 'gh-pages',
           commit: true,
           push: true
         }
       }
+    },
+
+    // Prompt to ask productioon configuration.
+    prompt: {
```
Comment (amitaibu) [inline app/templates/client/Gruntfile.js]: I prefer not to have prompts on the build process, so we can later use it in travis if we want.

Comment (ceoaliongroo) [inline app/templates/client/Gruntfile.js]: So, you recommend ask this information in the ``yo hedley``?

Comment (amitaibu) [inline app/templates/client/Gruntfile.js]: Don't ask at all - this should be configured manaualy.

**Comment by ceoaliongroo** [comment]
- The application is using the module config scripts minified 
screen shot 2015-01-22 at 11 02 35 am

- Generated by the config.json, property develpment
screen shot 2015-01-22 at 11 04 12 am

- Therefore, it's ignoring the config.js file with the production configuration.
screen shot 2015-01-22 at 11 04 35 am

**Comment by ceoaliongroo** [comment]
Close in favor PR #49

**Pull request closed without merging**